packages/components/src/components/hds/card/container.ts

```diff
@@ -129,14 +110,7 @@ export default class HdsCardContainer extends Component<HdsCardContainerSignatur
     return background;
   }
 
-  /**
-   * Sets the level for the card
-   * Accepted values: visible, hidden
-   *
-   * @param overflow
-   * @type {HdsCardOverflow}
-   * @default 'visible'
-   */
+  // Sets the level for the card
```

Comment:
@KristinLBradley this comment was not correct (it's actually setting the overflow). Maybe we could remove the comments that are redundant, and leave only the ones that are not obvoius?